js/lib/components/feedback/modal.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import Modal from "react-modal";
+import Highlighter from "web-highlighter";
+
+Modal.setAppElement("body");
+
+const modalStyles = {
+  content: {
+    top: "50%",
```

Comment:
Have you verified that this works (and looks good) on mobile?

Reply:
Here's what it looks like with the highlight and modal (albeit in chrome's device emulator):

https://user-images.githubusercontent.com/40175891/182518414-d4ea368e-0cdf-4c43-8057-41b320a2e484.mp4